demo/src/app/components/+rating/rating-section.list.ts

```diff
@@ -43,6 +44,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/custom/custom'),
         html: require('!!raw-loader?lang=markup!./demos/custom/custom.html'),
         outlet: DemoRatingCustomComponent
+      },
+      {
+        title: 'Select on enter',
+        description: 'Use this example for accessibility. After press Enter key the arrow keys are disabled.',
```

Comment:
please change the description because readonly datepicker disables all types of interaction, not only arrow keys